core/table.ts

```diff
@@ -112,6 +112,13 @@ export interface IBigQueryOptions {
    * For more information, read the [BigQuery partitioned tables docs](https://cloud.google.com/bigquery/docs/partitioned-tables).
    */
   partitionBy?: string;
+
+  /**
+   * The keys by which to cluster partitions by.
```

Comment:
@dwl285 Is this enough? Note the description for partitionBy is `The key with which to partition the table. Typically the name of a timestamp or date column.`

Reply:
LGTM